src/NzbDrone.Core/DecisionEngine/DownloadDecisionMaker.cs

```diff
@@ -64,14 +67,77 @@ private IEnumerable<DownloadDecision> GetMovieDecisions(List<ReleaseInfo> report
+                    //check if wanted language is found in Title String
+                    if (titleAndGroup[0].IndexOf(wantedLanguage) != -1)
+                    {
+                        //remove wanted language from movie title and place language after the year
+                        //titleAndGroup[0] = titleAndGroup[0].ToLower().Replace(wantedLanguage.ToLower(), "").Replace("..", ".");
+
+                        titleAndGroup[0] = Regex.Replace(titleAndGroup[0], wantedLanguage, "", RegexOptions.IgnoreCase);
+                        titleAndGroup[0] = titleAndGroup[0].Replace("..", ".");
+
+                        _logger.ProgressTrace("DEV: Language '{0}' has been removed: {1}", wantedLanguage, titleAndGroup[0]);
+
+                        string[] stringSeparatorsTemp = new string[] { yearInTitle };
+                        string[] titleTemp = titleAndGroup[0].Split(stringSeparatorsTemp, StringSplitOptions.RemoveEmptyEntries);
+                        titleAndGroup[0] = "";
+                        int i = 0;
+                        foreach (string ttemp in titleTemp)
+                        {
+                            _logger.ProgressTrace("DEV-Temp0: Array-Position : {0}", ttemp);
+                            if (i == 1) //because the year is the first seperator
+                            {
+                                titleAndGroup[0] = titleAndGroup[0] + yearInTitle + "." + wantedLanguage + ttemp;
+                                _logger.ProgressTrace("DEV-Temp1: New Title : {0}", titleAndGroup[0]);
+                            }
+                            else titleAndGroup[0] += ttemp;
+                            _logger.ProgressTrace("DEV-Temp2: New Title : {0}", titleAndGroup[0]);
+                            i++;
+                        }
+                        _logger.ProgressTrace("DEV: Language has been placed to the right order: {0}", titleAndGroup[0]);
+                    }
+                    else _logger.ProgressTrace("DEV: No Language found, seems to be an English RLS: {0}", titleAndGroup[0]);
+                }
+                else _logger.ProgressTrace("DEV: NO Year found in title: {0}", titleAndGroup[0]);
+
+
+                report.Title = titleAndGroup[0] + "-" + titleAndGroup[1];
+                _logger.ProgressTrace("DEV: Title after cleanup: '{0}'\n ...and now go processing:", report.Title);
+
                 DownloadDecision decision = null;
                 _logger.ProgressTrace("Processing release {0}/{1}", reportNumber, reports.Count);
 
                 try
                 {
-                    var parsedMovieInfo = Parser.Parser.ParseMovieTitle(report.Title);
+                    //var parsedMovieInfo = Parser.Parser.ParseMovieTitle(report.Title);
+                    //result from indexer
+                    var parsedMovieInfo = Parser.Parser.ParseMovieTitle(RemoveDiacritics(Parser.Parser.ReplaceGermanUmlauts(report.Title.Replace(":", " ").Replace(",", ""))));
```

Comment:
Same here, please move this into the Parser (Also Replace German Umlauts is already called anyways as well as the other functions).